package.json

```diff
@@ -13,62 +13,39 @@
         "access": "public"
     },
     "scripts": {
-        "extract-pot": "d2-i18n-extract -p src/ -o i18n/",
-        "generate-locale": "d2-i18n-generate -n 'ui-forms' -p ./i18n/ -o ./src/locales/",
-        "localize": "yarn extract-pot && yarn generate-locale",
-        "prebuild": "rm -rf ./build/* && yarn localize",
-        "build:commonjs": "BABEL_ENV=commonjs babel src --out-dir ./build/cjs --copy-files --verbose",
-        "build:modules": "BABEL_ENV=modules babel src --out-dir ./build/es --copy-files --verbose",
-        "build": "NODE_ENV=production yarn build:commonjs && NODE_ENV=production yarn build:modules",
-        "postbuild": "NODE_ENV=production yarn build-storybook",
-        "build-storybook": "build-storybook -c .storybook -o ./build/docs",
-        "start-storybook": "start-storybook --port 5000",
-        "prestart": "yarn localize",
-        "start": "yarn start-storybook",
-        "start:testing": "STORYBOOK_INCLUDE_TESTING=1 yarn start --ci --port $PORT",
-        "cypress:browser": "cypress open --config baseUrl=http://localhost:$PORT",
-        "cypress:ci": "NO_COLOR=1 cypress run -b $BROWSER --config baseUrl=http://localhost:$PORT",
-        "cypress:open": "PORT=12345 concurrently --kill-others -n storybook,cypress 'yarn start:testing' 'wait-on http-get://localhost:$PORT && yarn cypress:browser'",
-        "cypress:run": "PORT=12345 concurrently --kill-others --success first -n storybook,cypress 'yarn start:testing' 'wait-on http-get://localhost:$PORT && yarn cypress:ci'",
-        "test": "jest"
-    },
-    "husky": {
-        "hooks": {
-            "commit-msg": "d2-style commit check",
-            "pre-commit": "yarn test && d2-style js apply"
-        }
-    },
-    "cypress-cucumber-preprocessor": {
-        "nonGlobalStepDefinitions": true
+        "prebuild": "rm -rf ./build/*",
+        "build": "d2-app-scripts build",
+        "build:docs": "d2-utils-docsite build ./docs -o ./build/docs --jsdoc ./src --jsdoc-output-file api.md",
+        "build:demo": "EXTEND_ESLINT=true build-storybook -c .storybook -o ./build/docs/demo",
+        "start": "yarn start:demo",
+        "start:docs": "d2-utils-docsite serve ./docs -o ./build/docs --jsdoc ./src --jsdoc-output-file api.md",
+        "start:demo": "EXTEND_ESLINT=true start-storybook --port 5000",
+        "start:testing": "EXTEND_ESLINT=true STORYBOOK_INCLUDE_TESTING=1 start-storybook --port 5001 --quiet --ci",
+        "cy:open": "concurrently --kill-others -n storybook,cypress 'yarn start:testing' 'wait-on http-get://localhost:5001 && d2-utils-cypress open'",
```

Comment:
Side note - would be cool if we could move this concurrency and wait-on logic into the wrapper lib (and align it with the configuration of the Cypress GitHub Action)

Reply:
Yeah, plan is to add a switch, e.g. `d2 utils cypress run --wait-on <url>` to avoid this ugliness.